FirebaseFirestore.podspec

```diff
@@ -61,7 +61,12 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
       '"${PODS_TARGET_SRCROOT}" ' +
       '"${PODS_TARGET_SRCROOT}/Firestore/third_party/abseil-cpp" ' +
       '"${PODS_ROOT}/nanopb" ' +
-      '"${PODS_TARGET_SRCROOT}/Firestore/Protos/nanopb"'
+      '"${PODS_TARGET_SRCROOT}/Firestore/Protos/nanopb"',
+
+    # The nanopb pod (which is pulled in indirectly) sets these defs, so we
+    # must too.  (We *do* require 16bit (or larger) fields, so we'd have to set
+    # at least PB_FIELD_16BIT anyways.)
+    'OTHER_CFLAGS' => '-DPB_FIELD_32BIT -DPB_NO_PACKED_STRUCTS=1'
```

Comment:
Preprocessor definitions go in `GCC_PREPROCESSOR_DEFINITIONS` along with the others. Note that in that config entry you should strip the leading `-D`.